packages/coinstac-common/test/models/pipeline/runner/remote-pipeline-runner.js

```diff
@@ -40,7 +41,7 @@ test('RemotePipelineRunner::run - basic', t => {
     const pipelineRunStub = () => {
       // sinon stub is freaking out, so self-stub
       pipelineRunStub.called = true;
-      runner.pipeline.run.apply(runner.pipeline, arguments);
+      runner.pipeline.run(...arguments);
```

Comment:
Lambdas (arrow fns) lack lexical scope. `arguments` is going to point to something unexpected here.